database/database.go

```diff
@@ -165,8 +160,6 @@ func CreateNew(dbFile, feeXPub string, log slog.Logger) error {
 		return err
 	}
 
-	log.Info("Database initialized")
```

Comment:
This seems like a log you wouldn't want to lose.

Reply:
Per #465, creating the database will soon be in a new dedicated binary, so this is somewhat temporary/irrelevant.